solution_architecture.md

```diff
@@ -251,41 +255,108 @@ All exposure events are collected by the ENF internally and are split up into "E
 
-As mentioned before, the individual scan instances from the exposure windows are weighted according to the weight attached to the individual bucket. When those individual instances are summed up, they can be multiplied with a transmission risk value (which in turn is derived from the TRL described before). The result is one normalized exposure time per day. If those times are summed up, the overall risk can be determined, as shown in *Figure 16*.
+### Days Since Onset of Symptoms
 
-Note that the transmission risk level plays a special role in the above calculations: It can be defined by the app and be associated with each individual diagnosis key (i.e. specific for each day of an infected person) that is being sent to the server. It contains a value from 1 to 8, which can be used to represent a calculated risk defined by the health authority. As an example it could contain an estimate of the infectiousness of the potential infector at the time of contact and, hence, the likelihood of transmitting the disease. The specific values are defined as part of the [app](https://github.com/corona-warn-app/cwa-app-android/blob/master/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ProtoFormatConverterExtensions.kt) - a motivation of the parameter choices is found in the document [Epidemiological Motivation of the Transmission Risk Level](transmission_risk.pdf).
+The "days since onset of symptoms" (DSOS) is a relative numeric measure that indicates the infectiousness of a diagnosis key. The reference point is the date of onset of symptoms that a user can provide when uploading diagnosis keys.
+
+It is optional to provide information about the symptom onset and there different levels of granularity. This results in five different patterns for determining the DSOS of a diagnosis key and numeric values between -14 and 4000.
+
+The different patterns are:
+
+1. **No information** - the user declines to provide information about symptom onset
+
+    <details>
+    <summary>Figure 17: Determine DSOS for 'No information'</summary>
+
+    | ![Figure 17: Determine DSOS for 'No information'](images/solution_architecture/dsos-no-information.png "Figure 17: Determine DSOS for 'No information'") |
+    |:--:|
+    | **Figure 17: Determine DSOS for 'No information'**|
+
+    </details>
+
+2. **No symptoms** - the user indicates that he/she has no symptoms
```

Comment:
Use they to be gender neutral.

```suggestion
2. **No symptoms** - the user indicates that they have no symptoms
```

Reply:
@Ein-Tim I like the proposal of dropping _he/she_ but I'm not sure if _they_ is such a good replacement. How about just dropping the relative clause and rephrase it to _the user has no symptoms_?

Reply:
Sounds good!

Reply:
fixed with https://github.com/corona-warn-app/cwa-documentation/pull/845/commits/19341c3fef9edf172d14c7773f933263b71ddefc